probe-rs-cli-util/src/logging.rs

```diff
@@ -52,6 +64,38 @@ fn colored_level(level: Level) -> ColoredString {
     }
 }
 
+struct ShareableLogger(Logger);
+
+impl Log for ShareableLogger {
+    fn enabled(&self, metadata: &log::Metadata<'_>) -> bool {
+        metadata.level() <= self.0.filter()
+    }
+
+    fn log(&self, record: &Record<'_>) {
+        if self.enabled(record.metadata()) {
+            self.0.log(record);
+        }
+    }
+
+    fn flush(&self) {
+        self.0.flush();
+    }
+}
+
+impl SharedLogger for ShareableLogger {
+    fn level(&self) -> LevelFilter {
+        self.0.filter()
+    }
+
+    fn config(&self) -> Option<&simplelog::Config> {
+        todo!()
```

Comment:
Should this be implemented?

Reply:
Theoretically yes :D I should def. do that. In practice it works tho :)

Reply:
```suggestion
        None
```